### PR TITLE
Array - max product

### PR DESCRIPTION
**Pull request by DevratAgarwal**
## Checklist
- Solution is tested on sphere engine <br/>
    `Note:` Please copy only the solution part. Do not copy/paste the template code. That way, any issues with template code can be caught early on.
    - [ ] Author
    - [x] Reviewer
- PR title is correct - [PR title conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#title-for-a-pull-request)
    - [ ] Author
    - [x] Reviewer
- Folder/File naming is correct - [Naming conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#naming-of-files-and-folders)
    - [ ] Author
    - [x] Reviewer
- Coding conventions are followed - [Coding conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#coding-conventions)
    - [ ] Author
    - [x] Reviewer
- Test cases (action item for **reviewer**) <br/>
    - To be ensured by checking the problem on sphere-engine
    - If any test cases are missing, please add them - [how to add new test cases](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#adding-new-test-cases)
    - [x] There are at least 2 test cases
    - [x] There is at least 1 test case that is different from the sample test cases

### CODE REVIEW

File: solutions/arrays/max-product.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+let arr2=readLine().split(" ");
+for(let index=0;index<arrayOneSize;index++)
+{
+	arr1[index]=parseInt(arr1[index])
+}
+for(let index=0;index<arrayTwoSize;index++)
+{
+	arr2[index]=parseInt(arr2[index])
+}
+let maximumArrayOne=Math.abs(arr1[0]),maximumArrayTwo=Math.abs(arr2[0])
```
Comment (shivateja1729) [inline solutions/arrays/max-product.js:25]: @DevratAgarwal  declare the two variables `maximumArrayOne` and `maximumArrayTwo` in separate lines.

**Comment by sailesh-ramayanam** [comment]
@shivateja1729 Problem description states limits for length and elements. Please remove the limits. Some students give unnecessary importance to the limits.

**Comment by shivateja1729** [comment]
> @shivateja1729 Problem description states limits for length and elements. Please remove the limits. Some students give unnecessary importance to the limits.

@sailesh-ramayanam  removed the limits and added this "Note : Array length cannot be zero". In the problem statement, there is no result expected if length is zero.

**Comment by DevratAgarwal** [comment]
@shivateja1729 @sailesh-ramayanam changes are addressed

**Review by shivateja1729 — APPROVED** [review]

**Comment by prathyusha10x** [comment]
@sailesh-ramayanam please review the updated code the approve the PR by tomorrow EOD

**Pull request merged**